engine/src/test/scala/cromwell/SubFunctionWorkflowSpec.scala

```diff
@@ -0,0 +1,59 @@
+package cromwell
```

Comment:
I'd still say both simply because it's not yet automated like tyburn is. However that's in flight (waiting on devops) and centaur is already as feature-ful as tyburn